script.js

```diff
@@ -21,4 +22,189 @@ document.querySelectorAll('.nav-item').forEach(n =>
+        Image: 'card6',
+        link1: 'CSS',
+        link2: 'HTML',
+        link3: 'BootStrap',
+        live: '#',
+        source: '#',
+        img: 'assets/card6.png',
+    },
+];
+
+const postCards = document.querySelector('.post');
+
+postCards.innerHTML += `
+            <img src="${post_details[0].img}"
+             alt="Placeholder"
+             />
+            <div class="multi-post">
+                <h2>${post_details[0].title}</h2>
+                <p>
+                    ${post_details[0].description}
+                </p>
+                <ul>
+                    <li>${post_details[0].link1}</li>
+                    <li>${post_details[0].link2}</li>
+                    <li>${post_details[0].link3}</li>
+                </ul>
+                <button class="btn-orange" id="see-pro" type="button">See Project</button>
+            </div>
+`;            
+
+const card_details = document.querySelector('.project-card');
+
+for (let i = 1; i < post_details.length; i += 1){
+    const cards = `
+            <div class="card ${post_details[i].Image}">
+                <h1>${post_details[i].title}</h1>
+                <p>
+                    ${post_details[i].description}
+                </p>
+                <ul>
+                    <li>${post_details[i].link1}</li>
+                    <li>${post_details[i].link2}</li>
+                    <li>${post_details[i].link3}</li>
+                </ul>
+                <button class="btn-orange card-btn" id="project2" type="button">See Project</button>       
+            </div>
+    `;
+    card_details.innerHTML += cards; 
+}
+
+const popup = document.querySelector('.popup-container');
+
+ popup.innerHTML += `
+        <div class="overlay"></div>
+        <div class="popup">
+            <div class="popup-contain">
+                <div class="row justify-content-space-between">
+                    <h2 class="pop-title"></h2>
+                    <div class="close"><i class="fa-solid fa-xmark"></i></div>
+                </div>
+                <div class="row links">
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>BootStrap</li>
+                    </ul>
+                </div>
+                <div class="popup-content">
+                    <img src="" alt="Placeholder" class="pop-img">
+                    <div class="row column">
+                        <p class="pop-txt"></p>
+                        <div class="btn-div">
+                            <a class="btn-orange pop-btn" id="live" href="">See Live  <i class="fa-solid fa-globe"></i></a>
+                            <a class="btn-orange pop-btn" id="source" href="">See Source <i class="fa-brands fa-github"></i></a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    `;

```

Comment:
- [ ] You have done great so far :+1: :+1: But there are some changes you need to make before moving to the next milestone. :+1: 
- [ ] Currently, on a bigger screen above `1410 px`, your pop-up tends to not fit into the screen. Looking at the screenshot below, you can see that some parts are not visible. I'll suggest you adjust the height and with of this appropriately. :+1: 

- -  Screenshot:
![Screenshot from 2023-06-21 18-42-29](https://github.com/Mahdi-Niazi/Portfolio_Mobile_Setup/assets/107632780/5c8728ba-a0a2-4512-87dd-c3698b088c8e)

- [ ] Also on mobile screens, the content is squashed together, and there are no spacing and margins like the Figme design, I'll suggest you look into this too.

- - Your Current design:
![Screenshot from 2023-06-21 18-46-51](https://github.com/Mahdi-Niazi/Portfolio_Mobile_Setup/assets/107632780/2e0ffcbb-19be-4c6f-9c80-2b5e9259d460)

- -Figma design:
![Screenshot from 2023-06-21 18-47-20](https://github.com/Mahdi-Niazi/Portfolio_Mobile_Setup/assets/107632780/7e9da6c2-9ed8-4f01-9477-33796e7f71f0)